build/generate-deploy-index/index.js

```diff
@@ -25,17 +26,31 @@ ensureDirectory(STYLEGUIDE_DIST);
 
 const STYLEGUIDE_DEPLOYS = getDirectories(STYLEGUIDE_DIST).filter((d) => d !== '0.0.0-semantic-release');
 const LAB_DEPLOYS = getDirectories(LAB_DIST);
+const additionalVersions = ['latest', 'latest-preview'];
 
 const VERSION_REGEX = /^\d+\.\d+\.\d+/;
+function isNumericVersion(deployName) {
+	return VERSION_REGEX.test(deployName);
+}
 function isVersion(deployName) {
-	return ['latest', 'latest-preview'].includes(deployName) || VERSION_REGEX.test(deployName);
+	return additionalVersions.includes(deployName) || isNumericVersion(deployName);
```

Comment:
should probably do `semverValid(deployName)` here instead of `IsNumericVersion(deployName)`

Reply:
i think the only reason i didn't use wording like that is that semver seems to have its own [`valid()`](https://github.com/npm/node-semver/blob/master/functions/valid.js) function. is there any reason we're not using it?

Reply:
oh oops i just saw that u recommended using it lol